eo-maven-plugin/src/main/resources/org/eolang/maven/pre/to-java.xsl

```diff
@@ -131,6 +131,8 @@ SOFTWARE.
         <xsl:apply-templates select="//meta[head='package']" mode="head"/>
         <xsl:text>import org.eolang.*;</xsl:text>
         <xsl:value-of select="eo:eol(0)"/>
+        <xsl:text>import java.util.concurrent.atomic.AtomicBoolean;</xsl:text>
```

Comment:
@levBagryansky maybe this one we also should print only when it's a literal?

Reply:
@yegor256 Let's use fully qualified names then